### DISCUSSION TITLE
Always show rich text editor when available?

### DISCUSSION THREAD

**Discussion by jordan-brough** [chatwoot/chatwoot#5321] [Feature Requests]
This is mostly a question, and possibly a feature request depending on the answer.

Instead of having a button to turn the rich text editor on & off, why not just always show the rich text editor when a channel supports it?

We've had some confusion among our agents who didn't realize why the options weren't showing up and didn't realize they had to click the button to enable the rich text editor.

Are there some cases where rich text is supported but the rich text editor needs to be disabled? i.e. when you can't just "not use the rich text editor buttons"?

**Comment by pranavrajs** [comment]
cc: @nithindavid

**Comment by nithindavid** [comment]
@jordan-brough Yes we are aligned to this, But the current editor needs some changes to make it work it as plain text editor so it'll work on text only channels like whatsapp, SMS etc.

**Reply by jordan-brough** [reply]
What do you mean by "make it work as a plain text editor"?  My thought was to keep the current editors and experience, but just remove the "show rich text editor" button and always switch to the rich text editor in channels that support it.  Is that different than what you are talking about?

**Comment by pranavrajs** [comment]
@jordan-brough I think the suggestion is valid, and it makes sense to use the rich content editor only on the website and email channels. An option to toggle is not very useful. I've taken this up here. https://github.com/chatwoot/chatwoot/pull/5357
